docs/utils/NftokenTypes.ts

```diff
@@ -24,15 +24,15 @@ export namespace NftokenTypes {
 
   export const MetadataZ = z.object({
     name: z.string(),
-    description: z.string().nullable(),
+    description: z.string().nullable().optional(),
```

Comment:
I don't think we should change the metadata type. We can make it partial whenever we need less things.

Reply:
Partial is an "all or nothing" solution, what I wanted this schema to be useful for is to validate a CSV file which has to have values for `name` and `image` but can miss the rest of them, `nullable()` is not enough for that and using `.partial()` will render the whole validation useless because it will allow missing `name` and `image`.

What's the downside of `.optional()`? It seems to reflect the schema more correctly, doesn't it?

Reply:
I don't think we should update this to validate CSV parsing. If we want to have our own CSV validation, we can do that. If we change anything here, it should be discussed in a separate PR, not wrapped into a bigger PR.